### PR TITLE
Update media upload for Node.js discovery sample gen

### PR DESCRIPTION
**Pull request by tcoffee-google**
Initialize fields required by Node.js client libraries for media
upload, also needed to pass compile check.

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline [hunk 1]
```diff
@@ -1571,10 +1571,13 @@ google.auth.getApplicationDefault(function(err, authClient) {
...
 
-
-    // TODO: Add desired media content for upload. For more information, see:
-    // https://github.com/google/google-api-nodejs-client#media-uploads
-
+    media: {
+      // TODO: Change placeholders below to add desired media content for upload.
+      // For more information, see:
+      // https://github.com/google/google-api-nodejs-client#media-uploads
+      mimeType: '{MY-MIME-TYPE}',  // see https://www.w3.org/Protocols/rfc1341/4_Content-Type.html
```
Comment (saicheems) [inline src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline]: Would capitalize 'see'

**Comment by saicheems** [comment]
lgtm

File: src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline (hunk 1 above)
Comment (vchudnov-g) [inline src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline]: +1

Comment (tcoffee-google) [inline src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline]: Done

**Pull request merged**